src/Raven.Server/Utils/ExceptionHelper.cs

```diff
@@ -48,6 +53,11 @@ public static bool IsPageFileTooSmall(this Exception e)
             return e is Win32Exception win32Exception && win32Exception.NativeErrorCode == ERROR_COMMITMENT_LIMIT;
         }
 
+        public static void ThrowMediaIsWriteProtected(Exception inner)
+        {
+            throw new IOException(Sparrow.Server.Platform.PalHelper.ErrorMediaIsWriteProtectedHintMessage, inner);
```

Comment:
Let's include the original exception message as well. Out hint is just an addition.

```
throw new IOException($"{inner.Message}. {Sparrow.Server.Platform.PalHelper.ErrorMediaIsWriteProtectedHintMessage}", inner);